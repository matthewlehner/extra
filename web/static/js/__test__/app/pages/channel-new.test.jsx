// @flow
import React from "react";
import renderer from "react-test-renderer";

import ChannelNewPage from "app/pages/channel-new";

describe("ChannelNewPage component", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ChannelNewPage />);

    expect(tree).toMatchSnapshot();
  });
});
