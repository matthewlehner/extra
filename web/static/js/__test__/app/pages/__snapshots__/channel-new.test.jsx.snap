// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChannelNewPage component renders correctly 1`] = `
<div>
  <header
    className="heading"
  >
    <h1>
      Add a new channel
    </h1>
  </header>
  <section>
    <h2>
      Type
    </h2>
  </section>
  <section
    className="new-social-channel__section"
  >
    <h3
      className="social-logo social-logo_twitter"
    >
      <svg
        className="icon"
      >
        <use
          xlinkHref="#id"
        />
      </svg>
      <span>
        Twitter
      </span>
    </h3>
    <ul
      className="new-social-channel__ul"
    >
      <li
        className="new-social-channel__li"
      >
        <a
          className="button button_small button_twitter"
          href="/auth/twitter"
        >
          Add an account
        </a>
      </li>
    </ul>
  </section>
</div>
`;
